src/test/java/com/icthh/xm/ms/entity/service/tenant/provisioner/TenantDefaultUserProfileProvisionerTest.java

```diff
@@ -47,15 +47,20 @@ public void createTenant() {
     @Test
     public void manageTenant() {
         provisioner.manageTenant("NEWTENANT", "ACTIVE");
-        Mockito.verifyZeroInteractions(tenantContextHolder);
-        Mockito.verifyZeroInteractions(profileService);
+
+        Mockito.inOrder(profileService, tenantContextHolder).verifyNoMoreInteractions();
+//        Mockito.verifyZeroInteractions(tenantContextHolder);
+//        Mockito.verifyZeroInteractions(profileService);
```

Comment:
@osiievska23 check why it commented? For  deleteTenant()  to

Reply:
after migration
`Mockito.verifyZeroInteractions(tenantContextHolder);
 Mockito.verifyZeroInteractions(profileService);`

has changed to:
`Mockito.inOrder(profileService, tenantContextHolder).verifyNoMoreInteractions();`

I forgot to remove comments, thanks

Reply:
done